File: app/src/main/java/com/example/mike/mp3player/client/views/viewholders/EmptyListViewHolder.java
package com.example.mike.mp3player.client.views.viewholders;

import android.support.v4.media.MediaBrowserCompat;
import android.view.View;

import androidx.annotation.NonNull;

import com.example.mike.mp3player.client.AlbumArtPainter;

public class EmptyListViewHolder extends MediaItemViewHolder {
    public EmptyListViewHolder(@NonNull View itemView, AlbumArtPainter albumArtPainter) {
        super(itemView, albumArtPainter);
    }

    @Override
    public void bindMediaItem(MediaBrowserCompat.MediaItem item) {

    }
}

File: android/client/src/main/java/com/github/goldy1992/mp3player/client/views/viewholders/EmptyListViewHolder.kt
package com.github.goldy1992.mp3player.client.views.viewholders

import android.support.v4.media.MediaBrowserCompat
import android.view.View
import com.github.goldy1992.mp3player.client.AlbumArtPainter

class EmptyListViewHolder(itemView: View, albumArtPainter: AlbumArtPainter?) : MediaItemViewHolder(itemView, albumArtPainter) {
    override fun bindMediaItem(item: MediaBrowserCompat.MediaItem) {}
}